sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,15 +1,21 @@
 # Release History
 
-## 1.5.0-beta.3 (Unreleased)
+## 1.5.0 (2023-05-04)
 
 ### Features Added
 
-### Breaking Changes
+- Added support for challenge-based and multi-tenant authentication.
+- Added `DefaultAzureCredential`.
 
 ### Bugs Fixed
 
+- [[#4443]](https://github.com/Azure/azure-sdk-for-cpp/issues/4443) Fixed potentially high CPU usage on Windows.
```

Comment:
How is this an identity bug fix?

Reply:
Identity also had a call to `std::isalnum()`.